.github/workflows/tests.yaml

```diff
@@ -79,5 +79,7 @@ jobs:
           make
           make install
           psql -c 'CREATE EXTENSION pgtap;' || true
-      - name: Run Tests
-        run: pg_prove --username ${{ env.PGUSER }} --dbname ${{ env.PGDATABASE }} db/test/*/*_test.sql
+      - name: Run Database Unit Tests
+        run: pg_prove --username ${{ env.PGUSER }} --dbname ${{ env.PGDATABASE }} db/test/unit/**/*_test.sql
+      - name: Run Database Style Tests
+        run: pg_prove --username ${{ env.PGUSER }} --dbname ${{ env.PGDATABASE }} db/test/style/*_test.sql --set schemas_to_test=ccbc_public,ccbc_private
```

Comment:
the `db/test/style` folder is coming in via `.gitmodules` added in this PR but it looks like the checkout action isn't pulling in the git submodule. Perhaps an update to the checkout step?
```yaml
steps:
- name: Checkout repository and submodules
  uses: actions/checkout@v2
  with:
    submodules: recursive
```

Reply:
Thanks, this worked. Also I had to properly install the module with `git submodule add https://github.com/cas.... /db/test/style`.